contrib/tkn-image/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG GOLANG_VERSION=1.13.8
+ARG ALPINE_VERSION=3.11
+ARG UBUNTU_VERSION=19.10
+ARG FEDORA_VERSION=31
+
+FROM golang:${GOLANG_VERSION} as builder
+COPY . /go/src/github.com/tektoncd/cli
+WORKDIR /go/src/github.com/tektoncd/cli
+RUN make bin/tkn
+
+FROM alpine:${ALPINE_VERSION} as tkn-alpine
+COPY --from=builder /go/src/github.com/tektoncd/cli/bin/tkn /usr/bin
+
+FROM ubuntu:${UBUNTU_VERSION} as tkn-ubuntu
```

Comment:
do we really need multi-os images? why not just the alpine one?

Reply:
Because some people may want to use something else than `alpine` (something with a `glibc` instead of `musl`, …). It doesn't cost too much to build and publish those too :angel: (and reduce the need for the user to do it's own.

Question then is : what would be the default os for `latest` and `0.8.0` for ex ?

Reply:
so just use `ubuntu` or `fedora` and that would be the default? It may not cost much but it adds confusion....

Reply:
`tkn:latest` would be 0.8.0 alpine based
`tkn:0.8.0` would be 0.8.0 alpine based
`tkn:ubuntu` would be 0.8.0 ubuntu based (as of today)
`tkn:0.8.0-ubuntu` would be 0.8.0 ubuntu based
`tkn:fedora` would be would be 0.8.0 fedora base (as of today)
etc…

A bit like what `golang` does :wink:

Reply:
(the target name is not the final tag name :wink:)